src/TokenAirdrop.hs

```diff
@@ -113,3 +108,11 @@ group :: Int -> [a] -> [[a]]
 group n list
   | length list <= n = [list]
   | otherwise = let (xs, xss) = splitAt n list in xs : group n xss
+
```

Comment:
I copied this over from another project, we have a version of this in plutus-extra we could use. Wasn't sure if it is worth adding it in here or as part of a separate PR

Reply:
Gonna add this as a comment to #26 so we can handle it there.